packages/reactivity/__tests__/computed.spec.ts

```diff
@@ -339,7 +331,6 @@ describe('reactivity/computed', () => {
       _msg = msg.value
     })
 
-    items.value = [1, 2, 3]
```

Comment:
This extra assignment is part of what is being tested in this edge case. The test may pass either way, but it needs this line to test the right thing.